README.md

```diff
@@ -1,2 +1,24 @@
-# rails-blog-app
-This project is for developing a blog using rails
+# README
+
```

Comment:
- [ ] Please it will be nice to update your readme file to reflect the changes for this milestone. Checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and adapt it to your project, as this will make your readme file more informative and professional.

Reply:
Thanks, @Mihndim2020 for pointing out the readme file. I have updated it